### PR TITLE
NFT-1175 added new tx

### PR DESCRIPTION
**Pull request by bjartek**

### CODE REVIEW

**Review by sahilsingla905 — APPROVED** [review]
LGTM!

File: testnet/inituser.cdc [hunk 1]
```diff
@@ -30,6 +30,22 @@ transaction {
             )
         }
 
+        //run the buy without init first
```
Comment (louisguitton) [inline testnet/inituser.cdc:33]: Why buy? Is that copy pasted from somewhere? I see only initUser changes

**Review by louisguitton — APPROVED** [review]

File: testnet/inituser.cdc (hunk 1 above)
Comment (bjartek) [reply testnet/inituser.cdc:33]: That is a comment i wrote to explain why it happends

Comment (bjartek) [reply testnet/inituser.cdc:33]: Should i remove this comment or not? I do not have insight into what you approve and the client has to match.

Comment (louisguitton) [reply testnet/inituser.cdc:33]: no it's fine, we will submit for approval now.
I was confused by the wording and would have written something like this myself, but doesn't matter
```suggestion
        // handle the case where buy was ran without init first
```

**Pull request merged**